.vuestorefrontcloud/docs/default.conf

```diff
@@ -7,6 +7,10 @@ server {
   }
 
   location /v2/ {
+    if ($request_uri ~ ^/(.*)\.html(\?|$)) {
+        return 302 /$1;
+    }
```

Comment:
Is that condition necessary? What does it do? Maybe some comment would be useful 😄

Reply:
I thought so but I've tried it without it and seems to work